### PR TITLE
Feat : Added Home Page

### PR DESCRIPTION
**Pull request by rittik-ghosh**
Hi All,

I have created the home page for my ecommerce site - shoe-kart.

Please review and let me know if any changes are to be made.

Screenshot (15)
Screenshot (16)
Screenshot (17)

### CODE REVIEW

File: home.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+}
+.bold-subheading{
+    font-weight: 1000;
+    padding:1.5rem 0rem;
+}
+input.search-input {
+    min-width: 19rem;
+    border-radius: 4px;
+    display: block;
+    padding: 9px 4px 9px 40px;
```
Comment (anamkaa) [inline home.css:18]: You can prefer to use rem instead of pixels here. Other than that code looks clean.

File: home.html [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+
+        </div>
+    </div>
+
+    <footer class="footer-container centered-column">
```
Comment (vaibhav-mahalle) [inline home.html]: Nice and clean code structure. No changes are required as of now.

File: home.css [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    background: transparent url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' class='bi bi-search' viewBox='0 0 16 16'%3E%3Cpath d='M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z'%3E%3C/path%3E%3C/svg%3E") no-repeat 13px center;
+  }
+
+.search-input input{
+    min-width: 25rem;
+}
+.nav-category{
+    display: flex;
```
Comment (vaibhav-mahalle) [inline home.css:26]: you can reuse the code using utility classes.

File: home.css [hunk 4]
```diff
@@ -0,0 +1,122 @@
+body{
+    background-color: var(--st-white);
+}
+.flex-spacearound{
+    display: flex;
```
Comment (vaibhav-mahalle) [inline home.css:5]: same as below.

**Review by vaibhav-mahalle — APPROVED** [review]

File: home.css (hunk 3 above)
Comment (rittik-ghosh) [reply home.css:26]: Thanks vaibhav. I will use utility class

File: home.css (hunk 4 above)
Comment (rittik-ghosh) [reply home.css:5]: sure

File: home.html (hunk 2 above)
Comment (rittik-ghosh) [reply home.html]: Thanks

File: home.css (hunk 1 above)
Comment (rittik-ghosh) [reply home.css:18]: sure will make changes, thanks

**Pull request merged**